### DISCUSSION TITLE
Disable ANNOYING spell check button!

### DISCUSSION THREAD

**Discussion by AdrianoAz** [minbrowser/min#2388] [Ideas]
The wavy red lines of that speel checker are a pain to people read, it's could be very nice a disable button or something like this, mainly for people who write in several kind of languages or have short vision problems, please consider it, my very thanks.

**Comment by berkayyyilmaz** [comment]
Absolutely. I could not find any way to disable it. - June 2026

**Comment by ioc2e3** [comment]
Agree ...

Please add ( Disable spell check ) option ...

.
